### PR TITLE
New File Uploader - lithi.io

### PR DESCRIPTION
**Pull request by lithium720**
Added a new file uploader.
Lets try that again.

### CODE REVIEW

File: ShareX.UploadersLib/ShareX.UploadersLib.csproj [hunk 1]
```diff
@@ -657,6 +658,7 @@
     </EmbeddedResource>
     <EmbeddedResource Include="Forms\UploadersConfigForm.de.resx">
       <DependentUpon>UploadersConfigForm.cs</DependentUpon>
+      <SubType>Designer</SubType>
```
Comment (Jaex) [inline ShareX.UploadersLib/ShareX.UploadersLib.csproj]: I wonder what is this.

**Comment by Jaex** [comment]
Thanks.

**Comment by Jaex** [comment]
By the way don't you think having url like this:

https://i.lithi.io/Q47B.png

would be better than this:

https://lithi.io/i/Q47B.png

**Comment by lithium720** [comment]
Yes. This is something I intend on doing in the near future. When I have it working I just have to change the return URL from my API on my web server.

**Comment by DrDeTox** [comment]
Hi, I have downloaded the files for ShareX, what are the steps to install it? Thanks

**Pull request merged**